### DISCUSSION TITLE
Profile Achievements

### DISCUSSION THREAD

**Discussion by abbasxygum** [community/community#62681] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

How to get the Beta Profile achievements

**Comment by mehdiiwani** [comment, marked as answer]
https://github.com/4xmen/Get-Github-Achievements follow this link and you will get the guided steps for it

**Comment by github-staff** [comment]
Thanks so much for posting in our Community @abbasxygum. Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀
